### ISSUE TITLE
[JOSS] Recommended apt-get installation installs old version (mlpack 2.0.1)?

### ISSUE THREAD

**Issue by rasbt** [mlpack/mlpack#1406]
I followed the instructions recommended in the docs via apt-get, and installed it as follows (on Ubuntu 16.04 LTS)

```
sudo apt-get upgrade
sudo apt-get update
sudo apt-get install libmlpack-dev
```

When I then look at the files installed, via `dpkg -L libmlpack-dev`, I see that these list 

>  * This file is part of mlpack 2.0.1.

I then checked via `apt list --installed | grep libmlpack-dev`

and saw that it seems to be the old version indeed (`libmlpack-dev/xenial,now 2.0.1-1 amd64 [installed]`)

I am wondering if this is an issue with compatible versions for my system, or is the 3.0.1 version not available via apt-get, yet? If so, can the 3.0.1 version for this review currently only be installed from source?

**Comment by rasbt** [comment]
Related, the command-line program reference documentation refers to version 2.2.5 (http://www.mlpack.org/docs/mlpack-3.0.1/man.html):

> Below is a list of the command-line executables mlpack 2.2.5 provides, with links to the documentation for each executable. This documentation may also be accessed with the --help parameter or through the man pages provided with your distribution of mlpack.

Is this just a documentation bug or is the mlpack 3.0.0 functionality currently only available for the C++ library and language (e.g. Python) bindings?

**Comment by rcurtin** [comment]
Thanks for pointing out the issue---I fixed http://www.mlpack.org/docs/mlpack-3.0.1/man.html; it was an oversight during the release process.

For apt, we don't control the mlpack package in the repositories.  I can see that 3.0.1 has been pushed to the upstream Debian unstable repositories: https://packages.debian.org/sid/libmlpack-dev, but it hasn't made it to Ubuntu yet.  Some time ago I submitted a request for Ubuntu to update the mlpack package on older Ubuntu versions but I received no response.

**Comment by rasbt** [comment]
I see. I'd say it would be good to add a small note about that until it gets updated. Since Ubuntu is a pretty widely adopted distro these days, it could safe some users from some misunderstanding

**Comment by rcurtin** [comment]
I added a note to the README and the build guide that will be merged with #1421.

**Comment by rcurtin** [comment]
Since #1421 is merged, I'll go ahead and close this now.
